trace/HTTP.md

```diff
@@ -29,24 +29,28 @@ incoming requests should be a span kind of SERVER.
 
 Propagation is how SpanContext is transmitted on the wire in an HTTP request.
 
-Implementations MUST allow users to set their own propagation format and MUST provide an 
+Implementations MUST allow users to set their own propagation format and MUST provide an
 implementation for B3 at least.
```

Comment:
should be in a different PR if action taken as it would be surprising to see this as a side effect of the work titled "Implementations should provide a way to avoid undesirable propagation"